src/kubectl-helm-minikube/install.sh

```diff
@@ -135,6 +141,16 @@ if [ ${KUBECTL_VERSION} != "none" ]; then
     fi
     curl -sSL -o /usr/local/bin/kubectl "https://dl.k8s.io/release/${KUBECTL_VERSION}/bin/linux/${architecture}/kubectl"
     chmod 0755 /usr/local/bin/kubectl
+    if [ -e "/usr/local/bin/kubectl" ]; then
+        if grep -q "The specified key does not exist." "/usr/local/bin/kubectl"; then 
```

Comment:
Why do we need to do this? I'd expect `curl` to return a non-zero exit code which denotes it couldn't find artifacts?

Reply:
it shows the same exit code (12) in both the cases i.e. with correct version with which it could find the artifact and without correct version with which it could not. Though the file /usr/local/bin/kubectl showed the change when I did not use the correct version to fetch artifacts:
<?xml version='1.0' encoding='UTF-8'?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Details>No such object: kubernetes-release/release/v1.29.xyz/bin/linux/amd64/kubectl</Details></Error>

Reply:
No longer relevant